src/main/webapp/extensions/extensionsConfiguration.json

```diff
@@ -1,6 +1 @@
-{
-	"geppetto-default/ComponentsInitialization": true,
-	"geppetto-osb/ComponentsInitialization": false,
-	"geppetto-vfb/ComponentsInitialization": false,
-	"geppetto-neuron/ComponentsInitialization": false
-}
+{"geppetto-neuron/ComponentsInitialization": true}
```

Comment:
Please do not commit this file. It is changed locally during the installation process but better not to push it